--- origsrc/gnustep-make-2.2.0/FilesystemLayouts/fhs-system	2009-08-02 16:49:33.551607300 -0500
+++ src/gnustep-make-2.2.0/FilesystemLayouts/fhs-system	2009-08-02 16:27:42.299571600 -0500
@@ -76,16 +76,16 @@ GNUSTEP_LOCAL_DOC=/local/share/GNUstep/D
 GNUSTEP_LOCAL_DOC_MAN=/local/share/man
 GNUSTEP_LOCAL_DOC_INFO=/local/share/info
 
-GNUSTEP_USER_DIR_APPS=GNUstep/Applications
-GNUSTEP_USER_DIR_ADMIN_APPS=GNUstep/Applications/Admin
-GNUSTEP_USER_DIR_WEB_APPS=GNUstep/WebApplications
-GNUSTEP_USER_DIR_TOOLS=GNUstep/Tools
-GNUSTEP_USER_DIR_ADMIN_TOOLS=GNUstep/Tools/Admin
-GNUSTEP_USER_DIR_LIBRARY=GNUstep/Library
-GNUSTEP_USER_DIR_HEADERS=GNUstep/Library/Headers
-GNUSTEP_USER_DIR_LIBRARIES=GNUstep/Library/Libraries
-GNUSTEP_USER_DIR_DOC=GNUstep/Library/Documentation
-GNUSTEP_USER_DIR_DOC_MAN=GNUstep/Library/Documentation/man
-GNUSTEP_USER_DIR_DOC_INFO=GNUstep/Library/Documentation/info
+GNUSTEP_USER_DIR_APPS=.GNUstep/Applications
+GNUSTEP_USER_DIR_ADMIN_APPS=.GNUstep/Applications/Admin
+GNUSTEP_USER_DIR_WEB_APPS=.GNUstep/WebApplications
+GNUSTEP_USER_DIR_TOOLS=.GNUstep/Tools
+GNUSTEP_USER_DIR_ADMIN_TOOLS=.GNUstep/Tools/Admin
+GNUSTEP_USER_DIR_LIBRARY=.GNUstep/Library
+GNUSTEP_USER_DIR_HEADERS=.GNUstep/Library/Headers
+GNUSTEP_USER_DIR_LIBRARIES=.GNUstep/Library/Libraries
+GNUSTEP_USER_DIR_DOC=.GNUstep/Library/Documentation
+GNUSTEP_USER_DIR_DOC_MAN=.GNUstep/Library/Documentation/man
+GNUSTEP_USER_DIR_DOC_INFO=.GNUstep/Library/Documentation/info
 GNUSTEP_USER_CONFIG_FILE=.GNUstep.conf
-GNUSTEP_USER_DEFAULTS_DIR=GNUstep/Defaults
+GNUSTEP_USER_DEFAULTS_DIR=.GNUstep/Defaults
