--- origsrc/gnustep-make-2.4.0/configure.ac	2010-03-13 13:41:20.000000000 -0600
+++ src/gnustep-make-2.4.0/configure.ac	2010-08-11 20:05:51.557768100 -0500
@@ -164,20 +164,6 @@ AC_SUBST(CYGWIN)
 
 AC_EXEEXT
 AC_OBJEXT
-if test "$MINGW32" = yes; then
-  echo "hosted on mingw32 .."
-  export SHELL=sh
-  export CC=${CC:-gcc}
-  export AR=${AR:-ar}
-  export RANLIB=${RANLIB:-ranlib}
-  export DLLTOOL=${DLLTOOL:-dlltool}
-elif test "$CYGWIN" = yes; then
-  echo "hosted on cygwin .."
-  export CC=${CC:-gcc}
-  export AR=${AR:-ar}
-  export RANLIB=${RANLIB:-ranlib}
-  export DLLTOOL=${DLLTOOL:-dlltool}
-fi
 
 #--------------------------------------------------------------------
 # Find the binary and compile tools
@@ -1438,9 +1420,7 @@ AC_MSG_CHECKING(if the compiler supports
 # make because of the ':'.
 
 AUTO_DEPENDENCIES=""
-if test "$CYGWIN" = yes; then
-  AC_MSG_RESULT(no: autodependencies do not work on cygwin)
-elif test ! ${GCC} = "yes" ; then
+if test ! ${GCC} = "yes" ; then
   AC_MSG_RESULT(no: it's not gcc)
 else
   if test "${gs_cv_gcc_major_version}" -ge "3" >&5 2>&5; then
